django_intercom/templatetags/intercom.py

```diff
@@ -68,6 +54,21 @@ def intercom_tag(context):
     if 'request' not in context:
         return {"INTERCOM_IS_VALID": False}
 
+    default_user = {
+        "INTERCOM_IS_VALID": True,
+        "intercom_appid": INTERCOM_APPID,
+        "email_address": None,
+        "user_id": None,
+        "user_created": datetime.datetime.utcnow(),
+        "name": None,
+        "enable_inbox": INTERCOM_ENABLE_INBOX,
+        "use_counter": 'false',
+        "css_selector": INTERCOM_INBOX_CSS_SELECTOR,
+        "custom_data": '{}',
+        "company_data": '{}',
+        "user_hash": None,
```

Comment:
nit: I'd add the key `"anonymous": None,` here so the full output is defined at once and easier to see at first sight